src/main/java/core/basesyntax/exeptions/InvalidUserPasswordException.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.exeptions;
```

Comment:
There is a typo in the package name `exeptions`. It should be `exceptions` to adhere to standard naming conventions and ensure consistency across the project.